config/initializers/rails_admin.rb

```diff
@@ -1,7 +1,10 @@
 RailsAdmin.config do |config|
   ## == Devise/Warden Authorization using User #admin? method ==
   config.authorize_with do
-    redirect_to main_app.root_path unless !warden.user.nil? && warden.user.admin?
+    if !current_user.nil? && !current_user.admin?
```

Comment:
is `!current_user.` the same as the original `!warden.user.`?

Reply:
Yeah but I felt that we should change it to current_user for consistency sake. We never use warden.user anywhere else but here. so why use it?

Reply:
current user is just divises way of saying warden.user basically.

Reply:
@straleyb good deal